ml-agents/mlagents/trainers/torch/attention.py

```diff
@@ -97,7 +97,13 @@ def __init__(
             self.self_size = 0
         self.ent_encoders = torch.nn.ModuleList(
             [
-                LinearEncoder(self.self_size + ent_size, 1, embedding_size)
+                LinearEncoder(
+                    self.self_size + ent_size,
+                    1,
+                    embedding_size,
+                    kernel_init=Initialization.Normal,
+                    kernel_gain=(0.125 / embedding_size) ** 0.5,
```

Comment:
Is there a reference how you picked this gain? Could you add a comment and/or link?

Reply:
I'm following the initialization scheme from http://www.cs.toronto.edu/~mvolkovs/ICML2020_tfixup.pdf and gain from https://github.com/openai/multi-agent-emergence-environments/blob/master/ma_policy/layers.py#L89

Reply:
Added the reference